src/FFI.php

```diff
@@ -267,9 +268,11 @@ private static function init(): void
             $libraryPaths[] = $vipshome . "/lib/";
         }
 
-        if (PHP_OS_FAMILY === "OSX" ||
-            PHP_OS_FAMILY === "Darwin") {
-            $libraryPaths[] = "/opt/homebrew/lib/"; // Homebrew on Apple Silicon
+        if (PHP_OS_FAMILY === "OSX" || PHP_OS_FAMILY === "Darwin") {
+            // Homebrew on Apple Silicon
+            $libraryPaths[] = "/opt/homebrew/lib/";
+            // See https://github.com/Homebrew/brew/issues/13481#issuecomment-1207203483
+            $libraryPaths[] = "/usr/local/opt/vips/";
```

Comment:
It's a few years since I used homebrew on x86 macs, I don't remember :(

Reply:
```shell
$ uname -a                                                                                                                                    ✔  15:31:49  
Darwin xxxxxxxxxx.local 23.3.0 Darwin Kernel Version 23.3.0: Wed Dec 20 21:28:58 PST 2023; root:xnu-10002.81.5~7/RELEASE_X86_64 x86_64
$ brew install vips
.......
$ brew --prefix vips
/usr/local/opt/vips
$ readlink -f /usr/local/opt/vips                                                                                                                                   ✔ /usr/local/Cellar/vips/8.15.1_2
```

All I can say is that that worked for me.

In the beginning I was setting it in `.zshrc` with `export DYLD_LIBRARY_PATH="/usr/local/opt/vips/lib:$DYLD_LIBRARY_PATH"` but that of course only works when running stuff through the terminal (or similar) - and it seems the brew people gave up on making it work (I didn't follow the entire conversation, but you can check the link in the comment I've left). Of course if the comment merits a longer description, I'm fine with that.

Reply:
Oh interesting, I don't remember that, maybe it moved. Is there a link to libvips in `/usr/local/lib` as well?

Reply:
Looks like it 🤔 
```shell
ls /usr/local/lib/libv
libvips-cpp.42.dylib@  libvips.42.dylib@      libvmaf.3.dylib@       libvmaf.dylib@
libvips-cpp.dylib@     libvips.dylib@         libvmaf.a@ 
```

Reply:
Whoa, I can confirm that using `/usr/local/lib/` instead of `/usr/local/opt/vips/` works as well!

Reply:
Great! Yes, homebrew builds packages to paths like `opt/PACKAGENAME/libxxxx`, then during install symlinks them into the main homebrew lib area. That's why uninstall and reinstall is so fast -- it's just creating and removing the library symlinks.

So I think if we have `vips/lib` in there it might pick up even uninstalled libvipses. It's probably best to just use `/usr/local/lib`.